rust/chains/hyperlane-sealevel/src/validator_announce.rs

```diff
@@ -131,7 +129,7 @@ impl ValidatorAnnounce for SealevelValidatorAnnounce {
 
 // Copied from the validator-announce contract
 mod contract {
```

Comment:
Same goes for this `mod contract` here - can be removed now that we can import from the validator announce program directly